### ISSUE TITLE
[qBittorrent - mount cifs] Unable to apply new capability set

### ISSUE THREAD

**Issue by Marnalas** [alexbelgium/hassio-addons#5]
**Expected Behavior**
> [cont-init.d] 92-mounts: executing...
> [05:14:47] INFO: Mounting smb share...
> [05:14:47] INFO: Network Disks mounting.. //192.168**/pms
> [05:14:47] INFO: Mount //192.168**/pms
> Creating /storage/storagecifs
> [05:14:47] INFO: Success!
> [cont-init.d] 92-mounts: exited 0.

**Current Behavior**
> [cont-init.d] 92-mounts: executing... 
> [05:14:47] INFO: Mounting smb share...
> [05:14:47] INFO: Network Disks mounting.. //192.168**/pms
> [05:14:47] INFO: Mount //192.168**/pms
> Creating /storage/storagecifs
> Unable to apply new capability set.
> [05:14:47] WARNING: Protection mode is ON. Unable to mount external drives!
> [cont-init.d] 92-mounts: exited 0.

Unlike what the log says, the protection mode is disabled.

**Possible Solution**
As far as I can tell, this problem started appearing with the Core 2021.2.3. Same as with the Samba NAS addon problem you solved by adding a apparmor.txt. My guess would be to add the same file so the mount capability is added to the docker container running qbittorrent.

**Steps to Reproduce**
Start the addon with a networkdisks set up.

**Context (Environment)**
Rpi4
Core Version: core-2021.2.3
Supervisor Version: supervisor-2021.02.11
Operating System: Home Assistant OS 5.11
Addon version : 14.3.3.99202101191832-7248-da0b276d5ubuntu18.04.1-ls113

**Comment by alexbelgium** [comment]
Hi, thanks for the detailed analysis! Which addon is impacted? Thanks

**Comment by Marnalas** [comment]
Hi, my bad, I forgot the most obvious information. This issue concerns your qBittorrent addon. Thanks a lot for it by the way.

**Comment by alexbelgium** [comment]
Thanks! What is the ls of the current version? It is the number at end of the addon version : ls116?

Thanks!

**Comment by jonathanicq** [comment]
@alexbelgium 

Great timming, I've the same issue
Just reloaded your repository but the new version is not available yet.
I'm using the Is113

**Comment by alexbelgium** [comment]
That's why then, there is a ls116 since one or two days that includes the apparmor file. Shane that it is not visible for your, but lots of people have seen issues with their installations, including me, when updating to the latest home assistant version

**Comment by jonathanicq** [comment]
It's very weird because I've just added your repository today and it came
with the old version.

Maybe you could just re-create the version?

**Comment by Marnalas** [comment]
I'm using the ls113 too. Can't seem to be able to access the ls116. Uninstalling the addon -> removing your repo -> adding it again -> still only ls113.

**Comment by alexbelgium** [comment]
That's weird ls113 is several weeks old...

Please try something : go in the addon store, click Reload in the same menu that you add custom repos. The screen will go black, then addons will appear again.

Then check the supervisor log. You should have first the code of repo updates, the warnings if some addons have old values, and then the line saying that the refresh is completed. This looks like that. If you don't... Then it is an issue with the home assistant installation... 

Another way to check if repo is up-to-date :do you see the ubooquity addon in my repo on HA? I've added it not long ago. 

Third element to check : do other addons such as official or community versions update? 

Thanks! 

```
21-02-21 17:41:06 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository
21-02-21 17:41:06 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/alexbelgium/hassio-addons repository
21-02-21 17:41:12 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of HassOS SSH port 22222 Configurator
21-02-21 17:41:12 INFO (MainThread) [supervisor.store] Loading add-ons from store: 111 all - 0 new - 0 remove
21-02-21 17:41:12 INFO (MainThread) [supervisor.store] Loading add-ons from store: 111 all - 0 new - 0 remove
```

**Comment by jonathanicq** [comment]
@alexbelgium 

For your First Point
21-02-21 18:19:43 INFO (MainThread) [supervisor.store] Loading add-ons from store: 93 all - 0 new - 0 remove

For the Second
Yes, I do see the Ubooquity 
I've tried even your "Test" repo and there I can see a newer version but still not 116 but 114

For the Third
I don't have any adblocker (In fact I was planning to add it today, but I got focus on yours addon first :) )

**Comment by alexbelgium** [comment]
Thanks that's already a good news! Version 114 should already implement apparmor so that should be good :) that's really strange that it updates to an old version of the addons! Well, let me know if 114 solves the mount issue at least that's already that... If I remember well the 116 version only solves an issue with the installation of the VueTorrent custom ui, and allows smbv1...

**Comment by jonathanicq** [comment]
@alexbelgium 
I do have a lot of warning, but I do believe that is just because they are low rating 

21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Samba NAS
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'startup' with 'before' is deprecated. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'auto_uart' is deprecated, use 'uart'. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'devices' use a deprecated format, the new format uses a list of paths only. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'startup' with 'before' is deprecated. Please report this to the maintainer of FTP Backup
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Scrutiny
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Emby beta NAS
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Samba NAS
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'startup' with 'before' is deprecated. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'auto_uart' is deprecated, use 'uart'. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'devices' use a deprecated format, the new format uses a list of paths only. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Samba NAS
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'startup' with 'before' is deprecated. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'auto_uart' is deprecated, use 'uart'. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'devices' use a deprecated format, the new format uses a list of paths only. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'startup' with 'before' is deprecated. Please report this to the maintainer of FTP Backup
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Scrutiny
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Emby beta NAS
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Samba NAS
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'startup' with 'before' is deprecated. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'auto_uart' is deprecated, use 'uart'. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'devices' use a deprecated format, the new format uses a list of paths only. Please report this to the maintainer of RPi MySensor Gateway
21-02-21 18:22:57 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of RPi MySensor Gateway

**Comment by jonathanicq** [comment]
@alexbelgium 

Quickly question 
Do you know if would be possible to map an external drive that is already in use by another addon ?

I've two external hard-drive connect and shared by samba using dianlight addons
https://github.com/dianlight/hassio-addons/tree/master/sambanas

Since I can't connect to them by SAMBA, could I map them without remove it from the samba sharing with ls113 ?  

Or maybe I should use NextCloud from your repository, would then work ?

**Comment by alexbelgium** [comment]
Hi, I've converted file endings from crlf to lf, could your I please try
again to reload the repos?

Regarding your question, there is no issue mapping a same drive in
different addons. I had removed this option starting from ls114 to allow
the addon to have an higher security score, however it will work if you
disable protected mode. You need to specify the local name, probably sda1
or sdb1 depending on the slot used. I also use sambanas to map my disks to
smb (BTW, did you correctly received the update to nas8 Today?), then I
mount locally a second time the sda1 in my emby addon as direct mount is
faster than smb.

Nextcloud won't help : you can mount external,drives such as smb in
nextcloud with the external disk app, but you can't share it with other
addons...

Le dim. 21 févr. 2021 à 19:56, Jonathan Lima <notifications@github.com> a
écrit :

**Comment by Marnalas** [comment]
After reload :

> 21-02-21 19:28:04 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
> 21-02-21 19:28:51 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository
> 21-02-21 19:28:51 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/dianlight/hassio-addons repository
> 21-02-21 19:28:51 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/alexbelgium/hassio-addons repository
> 21-02-21 19:28:51 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/sabeechen/hassio-google-drive-backup repository
> 21-02-21 19:28:51 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/adamoutler/HassOSArgonOneAddon repository
> 21-02-21 19:28:51 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/home-assistant/addons repository
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Emby beta NAS
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Scrutiny
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Samba NAS
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'startup' with 'before' is deprecated. Please report this to the maintainer of RPi MySensor Gateway
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'auto_uart' is deprecated, use 'uart'. Please report this to the maintainer of RPi MySensor Gateway
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'devices' use a deprecated format, the new format uses a list of paths only. Please report this to the maintainer of RPi MySensor Gateway
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of RPi MySensor Gateway
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of ArgonOne Active Cooling
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Emby beta NAS
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Scrutiny
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Samba NAS
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'startup' with 'before' is deprecated. Please report this to the maintainer of RPi MySensor Gateway
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'auto_uart' is deprecated, use 'uart'. Please report this to the maintainer of RPi MySensor Gateway
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on config 'devices' use a deprecated format, the new format uses a list of paths only. Please report this to the maintainer of RPi MySensor Gateway
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of RPi MySensor Gateway
> 21-02-21 19:28:53 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of ArgonOne Active Cooling
> 21-02-21 19:28:53 INFO (MainThread) [supervisor.store] Loading add-ons from store: 86 all - 0 new - 0 remove
> 21-02-21 19:28:53 INFO (MainThread) [supervisor.store] Loading add-ons from store: 86 all - 0 new - 0 remove

I see ubooquity  but still ls113 though.

**Comment by jonathanicq** [comment]
@alexbelgium 

Thank you for sharing that
Yes, I got the NAS8 (it has a bigfix reported by me :) - I kind of took the day to work on HA)

I've reloaded and so far, no lucky 
I even removed the addon, removed the repository, restart the HA, add the repository and still 113

**Comment by alexbelgium** [comment]
OK, then the only solution I see is that I'll duplicate it is my other repo names hassio-addons-test. It will be the latest version...

**Comment by alexbelgium** [comment]
Oops wrong button

**Comment by Marnalas** [comment]
From what I get it's more of a problem with HA's supervisor than with your addon so clsing this issue wouldn't be wrong. I hope they work that. Thanks a lot for your efforts.

I'm gonna use your other repo then.

**Comment by alexbelgium** [comment]
Argh this is so frustrating that it fails without an error message, and showing files that are several weeks old! I've duplicated the qbittorrent addon under a new slug at https://github.com/alexbelgium/hassio-addons-test
This this it HAS to work ;-)

**Comment by Marnalas** [comment]
Yup, using https://github.com/alexbelgium/hassio-addons-test I get the ls116 version. Thank you !

Yes I was wondering about the versionning convention. Maybe it doesn't like letters, I don't know :S Never having created an addon myself I can only try to guess.

**Comment by Marnalas** [comment]
Don't know if that helps but I noticed that HA can't access the changelog of the qBittorrent addon stored in https://github.com/alexbelgium/hassio-addons.
With https://github.com/alexbelgium/hassio-addons-test, it's OK.

**Comment by alexbelgium** [comment]
Well the thing about naming convention is that it is based on the work of another team named linuxserver.io  
I use their image, and create the addon in a way that their image integrates within home assistant, and make links to allow configuration of some key components.
It allows two things : distribute very quickly addons, but more importantly that I can auto-update my addons (using the "updater" addon) so that whenever linuxserver makes an update my addon is automatically updated ! this allows to have always up-to-date addons (for example the bitwarden addon was updated the day the new version got out) compared to manually updated addons that can take several months to be updated (the bitwarden addon wasn't updated from june to around december of last year, breaking functions with apps).
However, to do that I need to keep the same nomenclature as them... Thus the long names :-)

**Comment by alexbelgium** [comment]
> Yup, using https://github.com/alexbelgium/hassio-addons-test I get the ls116 version. Thank you !
> 
> Yes I was wondering about the versionning convention. Maybe it doesn't like letters, I don't know :S Never having created an addon myself I can only try to guess.

You gave me an idea :-) I've updated the version from the main repo to a much shorter name, with the same content. Could you please test if it works? Thanks! :-)

**Comment by Marnalas** [comment]
I used "Reload" but the qBittorrent addon still indicates that "14.3.3.99202101191832-7248-da0b276d5ubuntu18.04.1-ls113" is its current version.

**Comment by jonathanicq** [comment]
@alexbelgium 
Same here
14.3.3.99202101191832-7248-da0b276d5ubuntu18.04.1-ls113

**Comment by jonathanicq** [comment]
@alexbelgium 
Another question (I'm not sure if is it releated or not but, there here goes) 

on 116
I've added the share as described on your documentation however after the download been complete I don't see the file into the volume 

(I've also opened the terminal and used mount to see if the volume was mapped and got no results)

That's is the log of the addons
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 01-envfile: executing... 
[cont-init.d] 01-envfile: exited 0.
[cont-init.d] 10-adduser: executing... 
-------------------------------------
          _         ()
         | |  ___   _    __
         | | / __| | |  /  \
         | | \__ \ | | | () |
         |_| |___/ |_|  \__/
Brought to you by linuxserver.io
-------------------------------------
To support LSIO projects visit:
https://www.linuxserver.io/donate/
-------------------------------------
GID/UID
-------------------------------------
User uid:    0
User gid:    0
-------------------------------------
[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 30-config: executing... 
chown: cannot access '/config/data/qBittorrent': No such file or directory
[cont-init.d] 30-config: exited 0.
[cont-init.d] 91-configuration.sh: executing... 
[21:02:01] INFO: Downloads can be found in <\\192.168.1.134\120gbssd>
[21:02:02] INFO: Whitelisted subsets will not require a password : localhost,127.0.0.1,172.30.0.0/16,192.168.0.0/16
[21:02:02] INFO: WEBUI username set to admin
[21:02:02] INFO: Default username/password : admin/adminadmin
[21:02:02] INFO: Configuration can be found in /config/qBittorrent
[cont-init.d] 91-configuration.sh: exited 0.
[cont-init.d] 92-mounts: executing... 
[21:02:03] INFO: Mounting smb share(s)...
[21:02:03] INFO: ... \192.168.1.134\120gbssd successfully mounted to /mnt/\192.168.1.134\120gbssd
[cont-init.d] 92-mounts: exited 0.
[cont-init.d] 99-custom-scripts: executing... 
[custom-init] no custom files found exiting...
[cont-init.d] 99-custom-scripts: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.


**QBITTORRENT.LOG**
(I) 2021-02-21T21:01:53 - UPnP / NAT-PMP support [OFF]
(N) 2021-02-21T21:02:03 - qBittorrent v4.3.3 started
(N) 2021-02-21T21:02:03 - Using config directory: /config/qBittorrent/
(I) 2021-02-21T21:02:03 - Trying to listen on: 172.30.33.7:6881
(N) 2021-02-21T21:02:03 - Peer ID: -qB4330-
(N) 2021-02-21T21:02:03 - HTTP User-Agent is 'qBittorrent/4.3.3'
(I) 2021-02-21T21:02:03 - DHT support [ON]
(I) 2021-02-21T21:02:03 - Local Peer Discovery support [ON]
(I) 2021-02-21T21:02:03 - PeX support [ON]
(I) 2021-02-21T21:02:03 - Anonymous mode [OFF]
(I) 2021-02-21T21:02:03 - Encryption support [ON]
(I) 2021-02-21T21:02:03 - IP geolocation database loaded. Type: DBIP-Country-Lite. Build time: Mon Feb 1 01:07:58 2021.
(N) 2021-02-21T21:02:03 - Using built-in Web UI.
(N) 2021-02-21T21:02:03 - Web UI translation for selected locale (en_US) has been successfully loaded.
(N) 2021-02-21T21:02:03 - Web UI: Now listening on IP: *, port: 8080
(I) 2021-02-21T21:02:03 - Successfully listening on IP: 172.30.33.7, port: TCP/6881
(I) 2021-02-21T21:02:03 - Successfully listening on IP: 172.30.33.7, port: UDP/6881
(N) 2021-02-21T21:02:03 - 'Tropa.de.Elite.2007.BRAZiLiAN.ENGSUB.DVDRip.XviD' restored.
(I) 2021-02-21T21:02:04 - Detected external IP: 188.37.112.69

**Comment by alexbelgium** [comment]
Hi, the share must be in the format : "//SERVER/SHARE"
Here you wrote : "\192.168.1.134\120gbssd" I believe

**Comment by Marnalas** [comment]
@jonathanicq for your information :
`mount` will return results depending on the docker container it's used in. So you should first use `docker ps` to get the name of the container running qBittorrent (last column, should something like addon_XXX_qbittorrent").
You can run commands in the container using `docker exec -it addon_XXX_qbittorrent /bin/bash` (replace XXX with correct value). Using `mount` should now show you coherent results. You can exit using... `exit` ;)

**Comment by jonathanicq** [comment]
@alexbelgium 
I've setup as you share and downloaded a file but still don't know where it went.

@Marnalas 
Thank you, i'll try that.

**Comment by alexbelgium** [comment]
It went in the download path that you mentioned in the addon configuration. If you wrote //192.168.1.134/120gbssd, the files will be within the container in /mnt/120gbssd. To access it from somewhere else, you need to mount again the smb share. You could check on your shared drive if the file is there.

If you are not happy where the file was downloaded, you can change the download path from within the WebUI and it will move the file.

If you want to download to a place accessible by other local containers, you need to download in the /media or /share folders.

**Comment by jonathanicq** [comment]
@alexbelgium 
As I've shared before, I have a external disk shared by SAMBA NAS from [dianlight](https://github.com/dianlight/hassio-addons/tree/master/sambanas)

from my laptop I can access it by \\192.168.1.134\120gdssd 
Once I use your addons, the share still ON so it does not stop the actual SMB (and I believe that it should not do it, right ?) 

So, I saved that path as the target for the download and I can see it on the WebUI
However, once the download is done I can't find the file 

PS.: I do have several files already there

**Comment by alexbelgium** [comment]
Your laptop is on windows I suppose - so the address is indeed \\192.168.1.134\120gdssd ; on linux the address will be //192.168.1.134/120gdssd

Once the drive is shared by SAMBA NAS, any addons can access it as long as the addon is started.

It is complex for me to troubleshoot without seeing your system. As I understand, you have set the network share to //192.168.1.134/120gdssd , and the download path to /smb/120gdssd ? Then it should work... Have you checked the nomenclature of the path? and by right clicking on the download, that it is indeed in the right folder?

To troubleshoot things like that, I usually install the [Portainer ](https://community.home-assistant.io/t/home-assistant-community-add-on-portainer/68836)addon. If you go into the advanced options and remove the addons from the hidden elements, it allows to gain console access into the running container and troubleshoot what is actually happening. For example, to do "ls -l /mnt/120gdssd" and check that the mounted files are indeed there

**Comment by Marnalas** [comment]
@jonathanicq 

> ssl: true
> certfile: fullchain.pem
> keyfile: privkey.pem
> whitelist: 'localhost,127.0.0.1,172.30.0.0/16,192.168.0.0/16'
> SavePath: /mnt/pms
> Username: **
> smbv1: false
> PUID: 0
> PGID: 0
> networkdisks: //192.168**/pms
> cifsusername: **
> cifspassword: **
> TZ: ''

If it can help you this is my config. I use it right now and it works. My files are correctly downloaded on the mounted partition.

I use windows 10 on my PC so my SMB is accessible via \\192.168**\pms through the file explorer (windows uses \ instead of /). On a linux based OS, the same SMB would be accessible with //192.168**/pms, that's why I set it up like this in the addon's config (home assistant OS is linux based).
As @alexbelgium said, the addon creates a pms directory (in my case)  in /mnt and then use it to mount the SMB partition. That's why I set up /mnt/pms as my default SavePath. When I download a new torrent I complete this path accordingly in the qBittorrent popup (/mnt/pms/movies for example). Right now I can check that my files are correctly downloaded from my Windows 10 PC using \\192.168**\pms\movies. If I wanted to check it in the terminal I would have to use the commands as following : 

`docker exec -it addon_XXX_qbittorrent_ha /bin/bash`
`ls /mnt/pms/movies`
`exit`

> Once I use your addons, the share still ON so it does not stop the actual SMB (and I believe that it should not do it, right ?)

With Home Assistant OS, addons run in there own docker container. It's **almost** like each addon runs on a separate machine/environment. That means that a drive mounted with one addon is not mounted for another one (except if you mounted it in /media or /share because these forlders are common to all docker containers).
I use Samba NAS like you and also this qBittorrent addon and a Plex Media Server addon. Both the qBittorrent and the plex addons have to mount the same drive and they can do so without breaking the mount of the other addon because they are "dockerised".

I hope this gives you a clue about what you should set up in your situation.

**Comment by Marnalas** [comment]
@ alexbelgium

I hope that what I wrote is correct. Feel free to correct me. I don't want to mislead anyone.

Should I close this issue ? The base problem is fixed. I am affraid that closing would prevent us from helping @jonathanicq and force him to open another issue. I am not used to github issue system so I might be wrong about that.

**Comment by alexbelgium** [comment]
It's solved!!! :-D

Please try a repo refresh and I'm pretty sure you'll see addon ls177 (updated this morning!)!

There was an hidden template file that referenced ls113...

**Comment by Marnalas** [comment]
Just reloaded and I can confirm that the current version is indicated as 14.3.3-ls117-rev2.
I'll uninstall the one from teh test repo and install the one from your main repo right now.

**Comment by alexbelgium** [comment]
> @ alexbelgium
> 
> I hope that what I wrote is correct. Feel free to correct me. I don't want to mislead anyone.
> 
> Should I close this issue ? The base problem is fixed. I am affraid that closing would prevent us from helping @jonathanicq and force him to open another issue. I am not used to github issue system so I might be wrong about that.

Thanks very much you're 100% right on what you said. And indeed using terminal to check inside addons is easier than portainer!

**Comment by Marnalas** [comment]
Installation finished. Everything runs smoothly ! Thanks a lot :)

Everything else is fixed too. The addon has the correct icon (didn't have one before) and the changelog is correctly showed.

**Comment by jonathanicq** [comment]
@Marnalas 
You rock man, really !
I was with my mindset on windows where the share is not visibly "mounted" 
I've changed the configuration to add the smb volume thorough the mnt path and it worked


@alexbelgium 
I've also downloaded the ls117-rev2 with success 

I do agre with @Marnalas  that we can close this ticket

Thank you All
